htdocs/core/lib/functions.lib.php

```diff
@@ -5349,11 +5349,15 @@ function complete_head_from_modules($conf,$langs,$object,&$head,&$h,$type,$mode=
 						if ($values[3]) $langs->load($values[3]);
 						if (preg_match('/SUBSTITUTION_([^_]+)/i',$values[2],$reg))
 						{
-							$substitutionarray=array();
-							complete_substitutions_array($substitutionarray,$langs,$object,array('needforkey'=>$values[2]));
-							$label=make_substitutions($reg[1], $substitutionarray);
+							$substitutionarray=head();
+							complete_substitutions_array($substitutionarray,$langs,$object,array('needforkey'=>$values[2]),'complete_head_from_modules');
```

Comment:
Why adding a parameter here ?
Default should be "completesubstitutionarray" as a the generic function. We should use this one in most cases so external module that bring this function will be called for most substitions.